package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-html-parser",
-  "version": "0.0.5",
+  "version": "0.1.0",
```

Comment:
Why the version is `0.1.0`?
I think there is no break change.
If you have some reason, could you explain that?

Reply:
I usually follow [SemVer](https://semver.org/spec/v2.0.0.html), and since I added user-facing functionality, this would be a minor bump. Most of the changes so far probably should have been minor bumps.